### PR TITLE
Adds explicit program/project

### PR DESCRIPTION
**Pull request by bwalsh**
This PR:
* Adds explicit program/project
* Adds .env file for postgres config
* Populates program and transaction_logs tables

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
 
+
+# backwards compatibility: dd is a synonym for program, default project to "simulated"
+program ?= $(dd)
+project ?= simulated
+
+include .env
+export $(shell sed 's/=.*//' .env)
+
+#   PG_HOST: aced-gen3.ddns.net
```
Comment (victorskl) [inline Makefile:11]: Would you mind remove all PG details lines 11-16 here, please.

**Review by victorskl — COMMENTED** [review]
Thanks @bwalsh Brain for the PR. 

- The ACED dictionary does look really great for our UMCCR dictionary development reference.
- I understood the intent using `.env`; so that we can switch to remote db. This use case is perfectly fine. Just wanted to note that I'll have few improvement follow up after this PR get merged.

Apart from PG host details, all look good to me.  I can squash merge it from my end after details have removed.

Cheers!

**Comment by bwalsh** [comment]
I hadn't intended to check in dictionary/aced.  You can find it here https://github.com/ohsu-comp-bio/aced-dictionary.git

**Comment by victorskl** [comment]
Thanks @bwalsh Brian for update. Noted about ACED. Sorry about delay, I was away with annual leave. Merging this now.

**Pull request merged**